alicloud/service_alicloud_nas.go

```diff
@@ -0,0 +1,106 @@
+package alicloud
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/nas"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+type NasService struct {
+	client *connectivity.AliyunClient
+}
+
+func (s *NasService) DescribeFileSystems(fileSystemId string) (fs nas.FileSystem, err error) {
+
+	args := nas.CreateDescribeFileSystemsRequest()
+	args.FileSystemId = fileSystemId
+	invoker := NewInvoker()
+	err = invoker.Run(func() error {
+		raw, err := s.client.WithNasClient(func(nasClient *nas.Client) (interface{}, error) {
+			return nasClient.DescribeFileSystems(args)
+		})
+		if err != nil {
+			if IsExceptedErrors(err, []string{InvalidFileSystemIDNotFound, ForbiddenNasNotFound}) {
+				return WrapErrorf(err, NotFoundMsg, AlibabaCloudSdkGoERROR)
+			}
+			return WrapErrorf(err, DefaultErrorMsg, fileSystemId, args.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		resp, _ := raw.(*nas.DescribeFileSystemsResponse)
+		if resp == nil || len(resp.FileSystems.FileSystem) <= 0 {
+			return GetNotFoundErrorFromString(GetNotFoundMessage("fileSystemId", fileSystemId))
```

Comment:
Please modify line 29 to `return WrapErrorf(Error(GetNotFoundMessage("File System", fileSystemId)), NotFoundMsg, ProviderERROR)`